src/SPH.cpp

```diff
@@ -638,15 +560,16 @@ void update_scalar_function(Float3* pos, Neighbor_Data* neighbor_data, float* sc
 			dz = pos->z[particle] - pos->z[neighbor_ind];
 
 			dist = sqrt(dx*dx + dy*dy + dz*dz);
-			q = dist / D_RAD;
+			q = dist / D_SEARCH_RANGE;
```

Comment:
1.0f/D_SEARCH_RANGE kan göras utanför for-loopen, tänk på att en div generellt kostar mer än en mul, och detta är något man inte kan lita på att compilatorn optimerar. Detta kan fixas på flera ställen